src/app/components/preloader/preloader.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PreloaderComponent } from './preloader.component';
+
+@NgModule({
+  declarations: [PreloaderComponent],
+  exports: [PreloaderComponent],
+  imports: [CommonModule],
```

Comment:
If you don't use anything from CommonModule, these is no reason to import it. CommonModule contains ngIf, ngFor, async pipe etc

Reply:
Thanks, fixed.